docs/content/concepts/layer/implementation/ssp/_index.md

```diff
@@ -66,6 +66,46 @@ The figure below expresses represents the portion of the OSCAL stack as it relat
+| `role-id ` <br />([XML](/reference/latest/system-security-plan/xml-reference/#/system-security-plan/system-implementation/user/role-id) &#124; [JSON](/reference/latest/system-security-plan/json-reference/#/system-security-plan/system-implementation/users/role-ids)) | Local SSP | `role` <br />([XML](/reference/latest/system-security-plan/xml-reference/#/system-security-plan/metadata/role) &#124; [JSON](/reference/latest/system-security-plan/json-reference/#/system-security-plan/metadata/roles)) | Token | A single target role must be found in the source. Role could be defined in local instance metadata. |
+<!--
+| [leveraged-authorization/party-uuid](/reference/latest/system-security-plan/xml-index/#/party-uuid) | Leveraged-Authorization | Party | UUID | A single target party must be found by the UUID, a pointer to the party that manages the leveraged system. |
+| [responsibility/@provided-uuid](/reference/latest/system-security-plan/xml-index/#/@provided-uuid) | Local SSP | Export/<br/>Responsibility | UUID | A single target control must be found in the source. Reference to a control, in the local SSP, that is exported as a customer responsibility for leveraging systems. |
+| [responsible-party/@role-id](/reference/latest/system-security-plan/xml-index/#/@role-id) | Local SSP | Role | Token | A single target role must be found in the source. System-characteristics/responsible-party/@role-id and inventory-item\*/responsible-party/@role-id are points of reference, so it is appropriate to enforce a local constraint. |
+| [responsible-party/party-uuid](/reference/latest/system-security-plan/xml-index/#/party-uuid) | Local SSP | Party | UUID | A single target party must be found in the source. System-characteristics/responsible-party/party-uuid and inventory-item\*/responsible-party/party-uuid are the points of reference, so they are appropriate to enforce a local constraint. |
+| [responsible-role/@role-id](/reference/latest/system-security-plan/xml-index/#/@role-id) | Local SSP, leveraged-authorization | Role | Token | A single target role must be found by the ID. |
+| [responsible-role/party-uuid](/reference/latest/system-security-plan/xml-index/#/party-uuid) | Local SSP, leveraged-authorization | Party | UUID         | A single target role must be found by the UUID. |
+-->
 ### Modeling Validation Information
 OSCAL is designed to allow capture relevant details related to independent validation of components. See the [Validation Modeling](/learn/tutorials/validation-modeling/) tutorial for details.
```

Comment:
OSCAL is designed to allow the capture of relevant ...

Reply:
Fixed